README.md

```diff
@@ -10,13 +10,26 @@ database versions, including Oracle.
 
 ## Quickstart
 
-Clone this repository somewhere. Also make sure you have docker and docker-compose installed.
+Clone this repository and ensure you have docker and docker-compose installed.
 
-Ensure that the `DJANGO_PATH` variable points to the root of the Django repo:
+### Environment
+
+You must set the `DJANGO_PATH` environment variable to the path of your local Django checkout.
+This can either be added to the `.env` file, or interactively set in your shell, like so:
 
 `export DJANGO_PATH=~/projects/django/`
 
-If you see a docker-compose warning about it not being defined followed by an error ensure that is defined in the shell you are using.
+You must also check the `.env` file (in the root of this repository) and ensure that the versions
+of both Python and and database(s) you wish to test against are properly configured. For example,
+Django 5.0 requires Python 3.10 (or later) and PostgreSQL 13 (or later), so if you wish to
+test against Django 5.0 with PostgreSQL, an appropriate `.env` file may be:
+
+```env
+PYTHON_VERSION=3.10
+POSTGRES_VERSION=13
+```
+
```

Comment:
If you want a section on Windows, maybe something like this 🤔 (I'm not the best at wording)

```suggestion
#### Windows

If your operating system is Windows, you will need to add the following to you `.env` file

* `PWD` set to the path of the root of this repository[^1].
* `DOCKER_DEFAULT_PLATFORM=linux/amd64` [^2]

[^1]: On Linux and macOS, but not of Windows, `PWD` refers to the local directory. 

[^2]: `apt-get` does not work on Windows. Setting `DOCKER_DEFAULT_PLATFORM=linux/amd64` enables the dependencies defined in the `Dockerfile` to be installed.